src/component/archive/ArchiveTable/ArchiveTable.tsx

```diff
@@ -0,0 +1,329 @@
+                                        style: {
+                                            flex: cell.column.minWidth
+                                                ? '1 0 auto'
+                                                : undefined,
+                                        },
+                                    })}
+                                    className={classes.cell}
+                                >
+                                    {cell.render('Cell')}
+                                </TableCell>
+                            ))}
+                        </TableRow>
+                    );
+                })}
+            </>
+        );
+    };
+
+    return (
+        <PageContent
+            isLoading={loading}
+            header={
+                <PageHeader
+                    title={`${
+                        inProject ? 'Project Features Archive' : 'Archived'
+                    } (${
+                        rows.length < data.length
+                            ? `${rows.length} of ${data.length}`
+                            : data.length
+                    })`}
+                    actions={
+                        <>
+                            <TableSearch
+                                initialValue={globalFilter}
+                                onChange={setGlobalFilter}
+                            />
+                        </>
+                    }
+                />
+            }
+        >
+            <ConditionallyRender
+                condition={!loading && data.length === 0}
+                show={<TablePlaceholder />}
+                elseShow={() => (
+                    <>
+                        <SearchHighlightProvider value={globalFilter}>
+                            <Table {...getTableProps()} rowHeight="standard">
+                                <SortableTableHeader
+                                    headerGroups={headerGroups as any}
+                                />
+                                <TableBody {...getTableBodyProps()}>
+                                    {renderRows()}
+                                </TableBody>
+                            </Table>
+                        </SearchHighlightProvider>
+                        <ConditionallyRender
+                            condition={
+                                rows.length === 0 && globalFilter?.length > 0
+                            }
+                            show={
+                                <TablePlaceholder>
+                                    No features found matching &ldquo;
+                                    {globalFilter}&rdquo;
+                                </TablePlaceholder>
+                            }
+                        />
+                    </>
+                )}
+            />
+        </PageContent>
+    );
+};
+
+function getColumns(onRevive: (feature: string) => Promise<void>) {
```

Comment:
Small nit, but we try to standardize where we can and usually use `const getColumns = () =>` as function notation.